libs/financial-aid/shared/src/data/postalCodes.ts

```diff
@@ -0,0 +1,186 @@
+export const postalCodes: number[] = [
```

Comment:
Is it on the roadmap to get these values dynamically from the source (Pósturinn or Þjóðskrá or who ever is the owner)?

Reply:
Yes it on the roadmap, this is a temporary mock for now ⌚